week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -7,18 +7,22 @@
   NOTE: only the names should be returned, not the means of transport.
 */
 
-function journeyPlanner() {
+function journeyPlanner(location, mode) {
+  let allLocations = location.filter(x => x.includes(mode));
+  let yourLocation = mo.map(x => x[0]);
```

Comment:
Does this code run correctly? How could you fix the error here?

Reply:
Oh, sorry. I write the wrong variable on line 11.
 It will be:
let yourLocation = allLocations.map(x => x[o]);